modules/compute/vm-instance/variables.tf

```diff
@@ -321,18 +321,18 @@ variable "placement_policy" {
     condition     = var.placement_policy == null ? true : try(keys(var.placement_policy), null) != null
     error_message = <<-EOT
     The var.placement_policy should be either unset/null or be a map/object with
-    fields: vm_count (number), availability_domain_count (number), collocation (string), max_distance (number).
+    fields: vm_count (number), availability_domain_count (number), collocation (string), max_distance (number), gpu_topology (string).
```

Comment:
This comment is for line 318 type = any.

Terraform supports optional since 1.5 version. So we should make use of that and change the type optional(map(....))